src/graphql/assets.ts

```diff
@@ -47,9 +47,16 @@ const AssetContentType = {
 export function updateAsset(
```

Comment:
generally about the `update` fn, do you think it would make sense to change them from 5 arguments to one object argument?